detox/test/e2e/setup.js

```diff
@@ -1,6 +1,7 @@
 const { device } = require('detox');
 
 beforeAll(async () => {
-  await device.selectApp('example');
-  await device.launchApp();
+  if (device._currentApp) { // HACK: currently there's no API to tell whether we are in multi-apps mode or not
```

Comment:
Need a better way to achieve that...

Reply:
@d4vidi, we need to agree upon a public API then. If we have a `device.selectApp()`, then should we have `device.currentApp` public getter?

Reply:
@d4vidi, here I am making sure you see my question — do you think we should add
```
get currentApp() { ... }
```
 as a public getter to Device?